chall07/mpernia-.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func time_elapse(start time.Time) int {
```

Comment:
The norm that we all use in Go is `PascalCase` (for exported functions, which can be used from outside) and` camelCase` (for functions that cannot be used from outside). This means that in this case, your function should be called `timeElapse` because it is a function from the main that you don't want to export.